setup.py

```diff
@@ -30,18 +30,16 @@
 
 readme = open('README.rst').read()
 
-setup_requires = [
-    'autosemver~=0.0,>=0.5.3',
-]
-
```

Comment:
could we maybe use this opportunity to move to Poetry?

Reply:
> additionally, there are currently many users of this library. I think we should update the `README` to reflect the current status of the project. It might be nice to keep doing PyPI releases if not too much work.

External users are using this?

Reply:
hmm, I'm not convinced about it, no one is checking the issues or the PRs. What's the point of releasing it? They can still use the old version if they want to and if someone complains we can add it in the future, for now especially this service is not designed to be used from others, is just an internal tool.

Reply:
OK, but I would at least put a disclaimer at the top of the README to explain the situation then.